packages/mediacenter/kodi-binary-addons/pvr.freebox/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="pvr.freebox"
-PKG_VERSION="2.2.0-Matrix"
-PKG_SHA256="4195031c9bc63077cf03ce445679e8ab9a6b508c6b28ec842c296ce925966631"
+PKG_VERSION="2.0.1-Leia"
```

Comment:
wrong branch ?

Reply:
Same tag used for both Leia & Matrix releases: https://github.com/aassif/pvr.freebox/releases